src/plone/restapi/serializer/portlet.py

```diff
@@ -335,6 +372,30 @@ def get_view_url(context):
     return name, item_url
 
 
+class NewsPortletRenderer(NewsRenderer):
+    def render(self):
+        items = []
+        news = self.published_news_items()
+        for new in news:
+            itemList = getMultiAdapter((new, self.request), ISerializeToJsonSummary)()
```

Comment:
``new`` as variable name should be replaced with ``brain``.

No need to do ``brain.getObject()``, getIcon should be already in the brain.